script/script-v0.0.1.schema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$id": "https://HPC-buildtest.github.io/schemas/script/script-v0.0.1.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Script",
+  "type": "object",
+  "properties": {
+    "description": {
+      "type": "string",
+      "description": "A description for the build recipe."
+    },
+    "maintainer": {
+      "type": "string",
```

Comment:
Shouldn't the maintainer field be a list instead of string.

So something like 

```
maintainer:
  - "First Last <first.last@example.com>"
 -  "John Doe <john.doe@example.com>"
```

at first i had in mind that this would follow the git commit format

Reply:
Good point! I'll update this to the config I'm working on locally (note that PR isn't ready for review yet, it's a draft)